unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChatController.cs

```diff
@@ -263,6 +263,9 @@ public void UpdateChannelSearchResults(string payload)
     public Channel GetAllocatedChannel(string channelId) =>
         channels.ContainsKey(channelId) ? channels[channelId] : null;
     
+    public Channel GetAllocatedChannelByName(string channelName) =>
+        channels.Select(x => x.Value).FirstOrDefault(x => x.Name == channelName);
```

Comment:
you can avoid 1 linq operation by doing: `channels.Values.FirstOrDefault(x => x.Name == channelName);`